app/src/main/java/net/skyscanner/backpack/demo/compose/IconsStory.kt

```diff
@@ -102,14 +100,6 @@ fun IconsStoryCompose() {
     val context = LocalContext.current
     val clipboardManager = LocalClipboardManager.current
 
-    val autoMirrorBackground = BpkTheme.colors.statusSuccessSpot
```

Comment:
no longer highlighting that an icon can be automirrored in the sample. As that's the only leftover usage for it it doesn't feel important enough as it's quite visible which ones rotate anyways. The automirror property was also technically visible to consumers which shouldn't really be the case